torch/storage.py

```diff
@@ -153,6 +186,9 @@ def _to(self, dtype):
             storage = storage.clone()
         return storage
 
+    def to(self, device: torch.device, non_blocking: bool = False, **kwargs) -> T:  # type: ignore[type-var] # noqa: E704
```

Comment:
```suggestion
    def to(self, *, device: torch.device, non_blocking: bool = False) -> T:  # type: ignore[type-var] # noqa: E704
```
See below for details

Reply:
done